examples/github-appcards/README.md

```diff
@@ -44,15 +56,108 @@ https://github.com/miroapp/app-examples/assets/10428517/e2c7b34a-e97d-453e-b64b-
 
-# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
+# Database Configuration <a name="database"></a>
+
+1. Create a database in Supabase. First you may need to create an Org.
+2. Go into your dashboard and into your project and to the `table editor`.
+3. Click on `Create a new table`
+4. Name this table `Auth` and add in the following columns, with the respective `Format` as shown in the screenshot below. This table will hold access_tokens to be able to call the Miro REST API to sync changes which happen in the GitHub project.
+5. Click on `Create a new table` and name this table `card-mapping` and add in the following columns, with the respective `Format` as shown in the screenshot below. This table will hold the app card ID from Miro and the GitHub issue ID along with the MiroUserId.
+6. Click on `Edit column` for the `miroUserId` in the `card-mapping` table, and then add in the following `Foreign Key Relation` as shown in the screenshot below.
+
+<img width="423" alt="supabase-foreign-key-config-2" src="https://github.com/miroapp/app-examples/assets/10428517/1f103542-79b4-47d8-973b-01c32d7a1113">
+
+7. Once you save the `Foreign Key Relation` your `miroUserId` from the `card-mapping` table should look something like the screenshot below. Click `Save`. We need this to be able to associate the miroUserId with a access_token so we can invoke the Miro REST API.
+
+<img width="733" alt="supabase-foreign-key-config" src="https://github.com/miroapp/app-examples/assets/10428517/9af8af58-f2cd-46db-a91f-d6c0d76b016e">
+
+One you are done, it should look like this. Click save.
+
+![supabase-foreign-key-config-summary](https://github.com/miroapp/app-examples/assets/10428517/3670da24-12be-4302-853f-0f24d45063b9)
+
+# ☁️ Netlify Configuration <a name="netlify"></a>
+
+The code in this repo contains three different functions which are meant to be serverless functions:
+
+- netlify/functions/authorize.js
+- netlify/functions/issues.js
+- netlify/functions/project-cards.js
+
+<b>[authorize.js](https://github.com/miroapp/app-examples/blob/main/examples/github-appcards/netlify/functions/authorize.js)</b> is going to run when you share your app with someone and then go through the OAuth flow.
+
+<b>[issues.js](https://github.com/miroapp/app-examples/blob/main/examples/github-appcards/netlify/functions/issues.js)</b> is going to run when you update the title or description of an issue. We have a GitHub action which will do this, defined in `.github/workflows/issues.yml`. You will need to update the [GitHub Action URL](https://github.com/miroapp/app-examples/blob/main/examples/github-appcards/.github/workflows/issues.yml#L11) to point to your deployed Netlify function. It should look something like `https://miro-github.netlify.app/.netlify/functions/issues`.
```

Comment:
Make the links relative, otherwise it will not work in every scenario (such as viewing a specific branch)

Reply:
good point!